specs/lfp-appeals-api-spec.json

```diff
@@ -0,0 +1,340 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Appeals API",
+    "description": "LFP Appeals API Documentation",
+    "version": "1.0.0"
+  },
+  "paths": {
+    "/companies/{company-id}/appeals": {
+      "get": {
+        "tags": [
+          "Appeal"
+        ],
+        "summary": "Get appeals by penalty reference",
+        "operationId": "getAppealsByPenaltyReference",
+        "parameters": [
+          {
+            "name": "company-id",
+            "description": "ID of the company filing the appeal",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "penaltyReference",
+            "description": "ID Reference of the penalty against the company",
+            "in": "query",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Appeals resource retrieved successfully",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/components/schemas/Appeal"
+                  }
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "Appeals not found"
+          },
+          "500": {
+            "description": "Internal server error"
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "Appeal"
+        ],
+        "summary": "Create a new appeal",
+        "operationId": "submitAppeal",
+        "parameters": [
+          {
+            "name": "ERIC-identity",
+            "in": "header",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "company-id",
```

Comment:
can this be changed to underscores `company_number` for consistency across our api's?